src/constants/systemGeos.ts

```diff
@@ -1,6 +1,9 @@
+export const SYSTEM_INTERNATIONAL_CODE = "xab";
+export const SYSTEM_INTERNATIONAL_NAME = "international";
+
 export const systemGeoCodes = [
   "xaa", // No Geography
-  "xab", // International
+  SYSTEM_INTERNATIONAL_CODE,
```

Comment:
Are we happy with this change affecting CPR/CCLW as well?

Reply:
That was the requirement as I understood it. Previously we were not displaying either "International" or "No Geography"

Reply:
Awesome - just wanted to check!